configs/openmrs/frontend_config/ozone-frontend-config.json

```diff
@@ -270,30 +270,12 @@
     ],
     "extensionSlots": {
       "patient-chart-summary-dashboard-slot": {
-        "add": ["attachments-overview-widget", "allergies-details-widget"],
-        "remove": ["vitals-overview-widget"]
-      },
-      "patient-chart-dashboard-slot": {
-        "add": ["nav-group#lifeStyle"],
-        "configure": {
-          "nav-group#lifeStyle": {
-            "title": "Life Style",
-            "slotName": "lifeStyle-nav-group-slot"
-          }
-        }
-      },
-      "lifeStyle-nav-group-slot": {
-        "add": ["dashboard#lifeStyle"],
-        "configure": {
-          "dashboard#lifeStyle": {
-            "title": "Life Style",
-            "slot": "lifeStyle-dashboard-slot",
-            "path": "lifeStyle"
-          }
-        }
-      },
-      "lifeStyle-dashboard-slot": {
-        "add": ["obs-by-encounter-widget#lifeStyle"],
```

Comment:
Removing these removes the dashboard, did you try refreshing?

Reply:
@rbuisson, the reason dashboard is still visible is because you might have tested it using Implementer tools, and that doesn't override the existing configuration which is being fetched from the `/config/ozone-frontend-config.json` and the existing config adds the dashboard. Once this PR is merged, the lifestyle dashboard will no longer exist